src/components/KBadge/KBadge.vue

```diff
@@ -50,26 +50,11 @@ import { ref, computed, watch, useAttrs, PropType } from 'vue'
 import KButton from '@/components/KButton/KButton.vue'
 import KIcon from '@/components/KIcon/KIcon.vue'
 import KTooltip from '@/components/KTooltip/KTooltip.vue'
-import type { BadgeAppearance, BadgeAppearanceRecord, BadgeShapeRecord, BadgeShape } from '@/types'
+import { BadgeAppearance, BadgeShape, appearances, shapes } from '@/types'
```

Comment:
why did you remove `import type` ?
```suggestion
import type { BadgeAppearance, BadgeShape, appearances, shapes } from '@/types'
```